docs/functions.rst

```diff
@@ -147,15 +147,18 @@ Example:
 ScaleWorkload
 -------------
 
-ScaleWorkload is used to scale up or scale down a Kubernetes workload.
+ScaleWorkload is used to scale up or scale down a Kubernetes workload. It
+also sets the original replica count of the workload as output artifact with
+the key ``originalReplicaCount``.
```

Comment:
Just wanted to confirm - it won't break older BP, right? I don't see any breaking changes.

Reply:
No it won't break older BP.